docker/centos6/Dockerfile

```diff
@@ -23,9 +25,14 @@ RUN yum -y -e1 -d1 install readline-devel \
                            sqlite-devel \
                            cyrus-sasl-devel.x86_64 \
                            mysql-devel \
-                           postgresql-devel \
+                           libpqxx-devel \
+                           gcc \
```

Comment:
gcc is already installed at this point.

Reply:
Thanks @razorsedge. Trying this by deleting gcc